iorgen/generator.py

```diff
@@ -25,6 +25,7 @@
 from iorgen.parser_ruby import gen_ruby
 from iorgen.parser_rust import gen_rust
 from iorgen.parser_scheme import gen_scheme
+from iorgen.parser_mtlg import gen_mtlg
```

Comment:
Why `mtlg` and not `metalang`? Also the import could go alphabetically sorted with the others.